ansible/roles/webserver-content/tasks/main.yml

```diff
@@ -34,41 +25,28 @@
     mode: 0775
   with_items:
     - "{{ biblebox_default_content_root }}"
-    - "{{ biblebox_config_root }}"
-
-- name: move default content into place
-  command: rsync -a /tmp/BibleBox-skeleton/content {{ biblebox_default_content_root }}
-
-- name: move default shared into place
-  command: rsync -a /tmp/BibleBox-skeleton/shared {{ biblebox_default_content_root }}
 
-# Trailing slash deliberately present... we want the contents of the config
-#  directory, but not the directory itself in the target.
-- name: move default config into place
-  command: rsync -a /tmp/BibleBox-skeleton/config/ {{ biblebox_config_root }}
-
-# This makes the config files executable but
-# it seems like the only other option is to separately
-# set permissions on directory in an additional task
-# Since the config files will probably be going away
-# I think this is fine for now.
-- name: update permissions on biblebox config files
-  file:
-    dest: "{{ biblebox_config_root }}"
+- name: Create placeholder index file
```

Comment:
That's correct. You can remove the task that creates the placeholder index file, and replace it with a task like we have for copying/synchronising the admin ui. PM on slack if you're unfamiliar with ansible - happy to talk you through it.